wardroom.json

```diff
@@ -1,10 +1,10 @@
 {
     "ami": {
-        "build_version": "91163cee43139fc3633180b7db607c505fcbbef6",
+        "build_version": "5d9e5c4661f8b8f20f0444b3fc3af12ba50d6d72",
         "distribution": "Ubuntu",
-        "distribution_release": "xenial",
-        "distribution_version": "16.04",
+        "distribution_release": "bionic",
+        "distribution_version": "18.04",
         "kubernetes_cni_version": "0.6.0-00",
-        "kubernetes_version": "1.11.5-00"
+        "kubernetes_version": "1.13.0-00"
```

Comment:
any reason 1.13.0 was chosen over 1.13.2?

Reply:
TBH I had just quickly gone to the k8s website itself thinking they'd list the most preferred/stable patch (which was 1.13.0).  I guess that was in error and should have just checked the github repo for releases. Want me to update this (and should I rerun tests if its just a point release difference?)

Reply:
I hate to have you do extra work, but getting the newest patch release usually includes security vulnerability fixes. Would it be pretty onerous to update?

Reply:
+1 on above, we should update to the latest patch release if possible

Reply:
+1 on bumping to the latest patch release.

Reply:
No problem; will update this. My mistake for missing the point releases.